contrib/docker/tests/docker-compose.test.yaml

```diff
@@ -61,3 +61,77 @@ services:
     depends_on:
       redis:
         condition: service_started
+
+  egress:
+    image: livekit/egress:v1.9.0
+    #restart: unless-stopped
+    ports:
+      - "8080:8080" # Egress API
+      - "8081:8081" # Egress Health
+      - "7980:7980" # Template port
+    labels:
+      - "traefik.enable=true"
+      - "traefik.http.routers.egress.rule=Host(`egress.workadventure.localhost`)"
+      - "traefik.http.routers.egress.entryPoints=web"
+      - "traefik.http.services.egress.loadbalancer.server.port=8080"
+      - "traefik.http.routers.egress-ssl.rule=Host(`egress.workadventure.localhost`)"
+      - "traefik.http.routers.egress-ssl.entryPoints=websecure"
+      - "traefik.http.routers.egress-ssl.service=egress"
+      - "traefik.http.routers.egress-ssl.tls=true"
+      - "traefik.http.routers.egress-ssl.tls.certresolver=myresolver"
```

Comment:
I don't think the egress needs to be exposed outside Docker at all. Because all requests are routed through Livekit server (see https://docs.livekit.io/home/self-hosting/egress/)


```suggestion
```